modules/microshift-low-latency-install-kernelrt-rhel-edge.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-low-latency.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-low-latency-install-kernelrt-rhel-edge_{context}"]
+= Installing the {op-system-rt-kernel} in a {op-system-ostree-first} image
+
+While the real-time kernel is not necessary for low latency workloads, using the real-time kernel can optimize low latency performance and is recommended. You can include it in a {op-system-ostree} image deployment.
+
+.Prerequisites
+* You have a Red Hat subscription enabled on the host that includes {op-system-rt-kernel}.
+* You are using the x86_64 architecture.
+* You configured `osbuild` to use the `kernel-rt` repository.
+//TODO insert link after Eliane's MR is published. Should be right around here: https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/composing_a_customized_rhel_system_image/creating-system-images-with-composer-command-line-interface_composing-a-customized-rhel-system-image#appending-a-kernel_creating-system-images-with-composer-command-line-interface
+
+[NOTE]
+====
+The following example blueprint includes references gathered from the previous steps required to configure low latency for a {microshift-short} cluster.
+====
+
+[IMPORTANT]
+====
+A subscription that includes the {op-system-rtk} must be enabled on the host used to build the commit.
+====
+
+.Procedure
+
+. Use the following example blueprint to install the real-time kernel in a {op-system-ostree} image:
++
+[source,text]
+----
+name = "microshift-low-latency"
+description = "RHEL 9.4 and MicroShift configured for low latency"
+version = "0.0.1"
+modules = []
+groups = []
+distro = "rhel-94"
+
+[[packages]]
+name = "microshift"
+version = "*"
+
+[[packages]]
+name = "microshift-greenboot"
+version = "*"
+
+[[packages]]
+name = "microshift-networking"
+version = "*"
+
+[[packages]]
+name = "microshift-selinux"
+version = "*"
+
+[[packages]]
+name = "microshift-low-latency"
+version = "*"
+
+[[packages]]
+name = "systemd-resolved"
+version = "*"
```

Comment:
Why is this necessary? If it's related to low-latency, let's add a comment in the blueprint explaining.

Reply:
Dunno, I just copied the thing from the tests :)
I git-blamed a bit and seems this was for some specific tests that are irrelevant here.

Reply:
So, let's remove this block. It's irrelevant for the current blueprint.